modules/soplex/7.1.2/overlay/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+# Description:
+# Soplex is a LP solver, a common tool in Operations Research.git
+
+cc_library(
+    name = "zstr",
+    hdrs = glob(["src/soplex/external/zstr/*.hpp"]),
```

Comment:
Looks good now, thanks!

Would `zstr` be a good contribution to the BCR? Depending on it as a module would avoid linker issues that may arise if a user of this module also brings in their own `zstr`.